src/config_brain_stats.js

```diff
@@ -62,6 +62,7 @@ brain.stats.add = function(path, newContent) {
     current = current[item];
   }
 
+  // Redundant _.merge automatically alters first object
```

Comment:
If it is redundant, just fix it (thx, for pointing out)

Reply:
Okay will fix it if I've got some spare time!